driver/compmisc.ml

```diff
@@ -30,10 +31,16 @@ let init_path ?(dir="") () =
     !Compenv.first_include_dirs
   in
   let exp_dirs =
-    List.map (Misc.expand_directory Config.standard_library) dirs in
-  Load_path.init (dir :: List.rev_append exp_dirs (Clflags.std_include_dir ()));
+    List.map (Misc.expand_directory Config.standard_library) dirs 
+  in
+  let dirs = 
+    if !Clflags.no_cwd then 
+      List.rev_append exp_dirs (Clflags.std_include_dir ()) 
+    else 
+      dir :: List.rev_append exp_dirs (Clflags.std_include_dir ()) 
+  in
```

Comment:
This can be simplified into

```ocaml
let dirs =
  (if !Clflags.no_cwd then [] else [dir])
  @ List.rev_append exp_dirs (Clflags.std_include_dirs ())
```

Reply:
it's done, thanks!